admin/release.py

```diff
@@ -326,18 +339,30 @@ def publish_docs_main(args, base_path, top_level):
 
 class UploadOptions(Options):
     """
-    Options for uploading packages.
+    Options for uploading artifacts.
     """
     optParameters = [
         ["flocker-version", None, flocker.__version__,
-         "The version of Flocker to upload packages for."],
+         "The version of Flocker to upload RPMs for."
```

Comment:
Now that the deb repo branch has been merged this is no longer correct - change it back to "packages".